net-vpn/i2p/i2p-2.1.0.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit java-pkg-2 java-ant-2 systemd
+
+DESCRIPTION="A privacy-centric, anonymous network"
+HOMEPAGE="https://geti2p.net"
+SRC_URI="https://files.i2p-projekt.de/${PV}/i2psource_${PV}.tar.bz2"
+
+LICENSE="Apache-2.0 Artistic BSD CC-BY-2.5 CC-BY-3.0 CC-BY-SA-3.0 EPL-1.0 GPL-2 GPL-3 LGPL-2.1 LGPL-3 MIT public-domain WTFPL-2"
+SLOT="0"
+
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="nls test"
```

Comment:
`test` should go into [JAVA_PKG_IUSE](https://wiki.gentoo.org/wiki/Java_Developer_Guide#Java_USE_flag_usage) between the `EAPI` and `inherit` lines.